File: test7C#/test7/Model/Burger/ChickenBurger.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace test7.Model.Burger
{
    class ChickenBurger : SimpleBurger
    {
        public ChickenBurger() : base("Chicken Burger", Convert.ToDecimal(5.99))
        {
            base.Filling(base.Filling() + ",Live chicken!");
        }
    }
}

File: test7Go/oop/models/burger/ChickenBurger.go
package burger

import (
	"fmt"
)

type ChickenBurger struct {
	SimpleBurger
}

func NewChickenBurger() ChickenBurger {
	simpleBurger := NewSimpleBurger("Chicken Burger", 5.99)
	simpleBurger.Filling = fmt.Sprintf("%s,Live chicken!", simpleBurger.Filling)

	return ChickenBurger{simpleBurger}
}
